store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -329,6 +344,19 @@ public CarbonWriterBuilder withCsvInput(Schema schema) {
     return this;
   }
 
+  /**
+   * to build a {@link CarbonWriter}, which accepts row in CSV format
+   *
+   * @param jsonSchema json Schema string
+   * @return CarbonWriterBuilder
+   */
+  public CarbonWriterBuilder withCsvInput(String jsonSchema) {
+    Objects.requireNonNull(jsonSchema, "schema should not be null");
+    this.schema = Schema.parseJson(jsonSchema);
```

Comment:
parse error must be notified. what is some other string passed instead of jsonString ?

Reply:
why this new method ? we already support withCsvInput(Schema schema), csdk cannot use it ?